components/markdownComponents/codepen.component.tsx

```diff
@@ -0,0 +1,23 @@
+interface CodePenProps {
+    pen: string;
+    title: string;
+    tab?: string;
+    height?: string;
+}
+
+export function CodePenComponent(props: CodePenProps) {
+    console.log(props);
+    return (
+        <div className="codepen-component">
+            <iframe
+                src={`https://codepen.io/team/codepen/embed/preview/${props.pen}?height=265&theme-id=dark&default-tab=${props.tab || "js,result"}`}
```

Comment:
Doesn't this URL need BabylonJS in it?

Reply:
Interesting. this works as expected as well :-) 
But changing it to BabylonJS works too. Changed.